### ISSUE TITLE
allow to filter features by their names

### ISSUE THREAD

**Issue by kmike** [TeamHG-Memex/eli5#65]
Sometimes it is useful to check coefficients only for some of the features. For example, [here](https://github.com/TeamHG-Memex/autopager/blob/master/notebooks/Training.ipynb) (scroll down to "What are important features?") one may want to check how e.g. query:... features affect the result, without looking at all other features. This also can be helpful when adding a new feature.

What about adding 'feature_re' or 'feature_patterns' argument to explain_weights functions?

**Comment by lopuhin** [comment]
Yeah, that's useful! Let me add this.
